ligo/haskell/test/Test/Ligo/BaseDAO/OffChainViews.hs

```diff
@@ -7,23 +7,35 @@ module Test.Ligo.BaseDAO.OffChainViews
 
 import Universum hiding (view)
 
+import qualified Data.Map as M
+import Lorentz (BigMap(..))
 import Named (defaults, (!))
 import Test.Tasty (TestTree)
 
 import Tezos.Address
 
+import BaseDAO.ShareTest.Common (getTotalSupplyFromLedger)
 import BaseDAO.ShareTest.OffChainViews
 import qualified Ligo.BaseDAO.Types as DAO
 import qualified Ligo.BaseDAO.TZIP16Metadata as DAO
 import qualified Lorentz.Contracts.BaseDAO.TZIP16Metadata as DAO
 
 defaultStorage :: DAO.StorageL
 defaultStorage =
-  DAO.mkStorageL
-  ! #admin (unsafeParseAddress "tz1M6dcor9QNTFr9Ri68cBYvpxrogZaMttuE")
+  (DAO.mkStorageL
+  ! #admin addr
   ! #extra DAO.dynRecUnsafe
   ! #metadata mempty
   ! defaults
+  ) { DAO.sLedger = bal
+    , DAO.sTotalSupply = getTotalSupplyFromLedger bal
+    }
+  where
+    addr = unsafeParseAddress "tz1M6dcor9QNTFr9Ri68cBYvpxrogZaMttuE"
+    bal = BigMap $ M.fromList $
+      [ ((addr, DAO.unfrozenTokenId), 100)
+      , ((addr, DAO.frozenTokenId), 200)
+      ]
```

Comment:
Why the addition of this address balance(s) here?

I'd imagine that in a `defaultStorage` the ledger should be empty (as it was before)

Reply:
This `defaultStorage` is used to test FA2 off-chain views:

```
  , testGroup "get_total_supply" $
    let checkTotalSupply st param =
          runView @Natural (DAO.getTotalSupplyView mc) st (ViewParam param)
    in
    [ testCase "Get unfrozen token total supply" $
        checkTotalSupply storage DAO.unfrozenTokenId
          @?= Right 100
    , testCase "Get frozen token total supply" $
        checkTotalSupply storage DAO.frozenTokenId
          @?= Right 200
    ]
```

Maybe renaming this to something like offchainViewStorage would be better.